public/js/main.js

```diff
@@ -36,19 +37,27 @@ const actions = bindActionCreators(require("./actions"), store.dispatch);
 // global for debugging purposes only!
 window.store = store;
 
-connectClient(response => {
-  actions.newTabs(response.tabs);
+connectClient(tabs => {
+  actions.newTabs(tabs);
 
   // if there's a pre-selected tab, connect to it and load the sources.
   // otherwise, just show the toolbox.
   if (hasSelectedTab()) {
     const selectedTab = getSelectedTab(store.getState().tabs.get("tabs"));
-    debugTab(selectedTab.toJS(), actions).then(renderToolbox);
+    const tab = selectedTab.get("firefox") || selectedTab.get("chrome");
```

Comment:
Same as my comment above, I don't love this ambiguity because I'll find myself with the same page open in Firefox or Chrome (while testing) and it's not clear which browser I'm testing. I'd rather encode the target browser in the URL params.